### ISSUE TITLE
[Enhancement] Inform users that root-mode may require "SELinux Permissive".

### ISSUE THREAD

**Issue by ImPat** [brevent/Brevent#70]
I recently reset my SM-T810 and tried to install brevent on it. When I tried to launch brevent server with root mode and nothing happened, I started troubleshooting and eventually came to the decision that the problem must be caused by the kernel. Unfortunately, every nougat kernel I tried had the same issue and it didn't work. Eventually, I found a magisk module that promises to set SE for Android to permissive. And it worked, brevent server launched without an issue. 

Tldr:
SELinux Permissive may be required for root mode.

I suggest that you inform the user about this.

**Comment by liudongmiao** [comment]
Root mode is experimental, and I have said it may not work well.

It's unsafe to set Selinux to permissive. And the implementation of ROOT should be blamed.

Brevent use a lot of private api, some of them don't work with some ROOT. You should paste the log, and I will explain what's needed.

And for me, I don't use ROOT, and ROOT mode of Brevent doesn't offer more feature than normal ADB. 

Possible issues, root cannot run `dumpsys`, you can try it:
```
dumpsys battery
```
